dockerfiles/dockerfile.javadocmavenrepo

```diff
@@ -0,0 +1,5 @@
+ARG baseVersion
+ARG dockerRepository
+FROM ${dockerRepository}/galasadev/galasa-base:${baseVersion}

```

Comment:
Please can you update this to `FROM ${dockerRepository}/galasa-dev/base-image:${baseVersion}` as this is where the new base image in GHCR is.

Reply:
done